frontend/pages/inventory.tsx

```diff
@@ -1,9 +1,147 @@
+    }, [currentPage]);
+
+    const handlePageClick = (page: number) => {
+        setCurrentPage(page);
+    };
+
+    const renderPageNumbers = () => {
+        const pageNumbers = [];
+        for (let i = 1; i <= totalPages; i++) {
+            pageNumbers.push(
+                <li key={i} className={currentPage === i ? 'active' : ''}>
+                    <a href={`?page=${i}`} onClick={(e) => { e.preventDefault(); handlePageClick(i); }}>
+                        {i}
+                    </a>
+                </li>
+            );
+        }
+        return pageNumbers;
+    };
+
+    return (
+        <InventoryLayout>
+            <table className="min-w-full bg-white border border-gray-300 mb-4">
+                <thead>
+                    <tr>
+                        <th className="py-3 px-6 bg-gray-300 text-center">ID</th>
+                        <th className="py-3 px-6 bg-gray-300 text-center">Name</th>
+                        <th className="py-3 px-6 bg-gray-300 text-center">Description</th>
+                        <th className="py-3 px-6 bg-gray-300 text-center">Quantity</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {currentItems.map((item) => (
+                        <tr key={item.id} className="h-20 overflow-hidden">
+                            <td className="py-2 px-4 border-b text-center">{item.id}</td>
+                            <td className="py-2 px-4 border-b text-center">{item.name}</td>
+                            <td className="py-2 px-4 border-b text-center">{item.description}</td>
+                            <td className="py-2 px-4 border-b text-center">{item.quantity}</td>
+                        </tr>
+                    ))}
+                </tbody>
+            </table>
+            <ul className="pagination flex gap-2">
+                <li className={currentPage === 1 ? 'disabled' : ''}>
+                    <a
+                        href={`?page=${currentPage - 1}`}
+                        onClick={(e) => {
+                            e.preventDefault();
+                            if (currentPage > 1) {
+                                handlePageClick(currentPage - 1);
+                            }
+                        }}
+                    >
+                        &laquo; Prev
+                    </a>
+                </li>
+                {renderPageNumbers()}
+                <li className={currentPage === totalPages ? 'disabled' : ''}>
+                    <a
+                        href={`?page=${currentPage + 1}`}
+                        onClick={(e) => {
+                            e.preventDefault();
+                            if (currentPage < totalPages) {
+                                handlePageClick(currentPage + 1);
+                            }
+                        }}
+                    >
+                        Next &raquo;
+                    </a>
+                </li>
```

Comment:
should be disabled if last page (the `disabled` class won't do it)

Reply:
why wouldn't it? because in practice it works well

Reply:
In pratice it works but people navigating by keyboard (motor impaired, for example) can focus this `<a>` and activate it but nothing will happen.
Focus and activation are not impossible if using the `disabled` prop